src/workspaces/features/ArchiveWorkspaceDialog/ArchiveWorkspaceDialog.tsx

```diff
@@ -54,20 +55,30 @@ const ArchiveWorkspaceDialog = (props: ArchiveWorkspaceDialogProps) => {
   };
 
   return (
-    <Dialog open={open} onClose={onClose}>
+    <Dialog open={open} onClose={onClose} onClick={(e) => e.stopPropagation()}>
       <Dialog.Title>
         {t("Archive {{name}}", { name: workspace.name })}
       </Dialog.Title>
       <Dialog.Content className="space-y-4">
         <p>
           {t("You're about to archive this workspace and all its content.")}
         </p>
+        <p>{t("Please enter the workspace name to confirm:")}</p>
```

Comment:
I would at least put the name of the workspace in the sentence. Bonus: having it as a placeholder of the input below

Reply:
Good idea, thanks, change dit

Reply:
don't know If you are aware of it but we have a nice `<Trans/>` component that allow you to translate jsx/html (and thus, keep styles to put some items in emphase)

Reply:
The logic is that it takes the raw string and replace all tags by numeroted placeholders <0></0> and do the interpolation when it displays the translation. Pretty neat.